Library/Tests/ViewModels/RewardPledgeViewModelTests.swift

```diff
@@ -40,6 +40,7 @@ internal final class RewardPledgeViewModelTests: TestCase {
   fileprivate let differentPaymentMethodButtonHidden = TestObserver<Bool, NoError>()
   fileprivate let dismissViewController = TestObserver<(), NoError>()
   fileprivate let estimatedDeliveryDateLabelText = TestObserver<String, NoError>()
+  private let estimatedFulfillmentStackViewHidden = TestObserver<Bool, NoError>()
```

Comment:
Yeah my approach has been to make everything `private` and to explicitly change to `fileprivate` if it needs to be accessed by an extension or something.

Up to you!